applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/cr_beam_adjoint_element_3D2N.hpp

```diff
@@ -6,11 +6,10 @@
 //  License:     BSD License
```

Comment:
just a minor comment: if you say BSD license, it means nothing since there are different versions of it.

the kratos one is the 4-clause license. We make a pointer to the license file to have in there the entire text